docs/entities/index.md

```diff
@@ -22,8 +22,8 @@ The relationship between `EntityType`s and `Entity`s is similar to that of [`Ite
 Let's create our `EntityType` registry and register an `EntityType` for it, assuming we have a class `MyEntity` that extends `Entity` (see [below][entity] for more information). All methods on `EntityType.Builder`, except for the `#build` call at the end, are optional.
 
 ```java
-public static final DeferredRegister<EntityType<?>> ENTITY_TYPES =
-    DeferredRegister.create(Registries.ENTITY_TYPE, ExampleMod.MOD_ID);
+public static final DeferredRegister.Entities ENTITY_TYPES =
+    DeferredRegister.createEntities(ExampleMod.MOD_ID);
 
 public static final Supplier<EntityType<MyEntity>> MY_ENTITY = ENTITY_TYPES.register(
```

Comment:
This example should actually make use of the specialized DR's `registerEntityType()` methods.